.gitignore

```diff
@@ -14,7 +14,6 @@ gen/
 
 # Gradle files
 .gradle/
-gradle/
```

Comment:
@davideas I believe that this line should be removed to allow the wrapper to be included, but I just want to check that this won't break anything. Gradle says to include the wrapper files in repositories [here](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.wrapper.Wrapper.html), and there are already versions of the file in the repository; the problem is, they aren't updated by a simple `git push` after changing one of them.